api/src/main/java/org/datacleaner/api/UsageMeteringMessage.java

```diff
@@ -20,26 +20,44 @@
 package org.datacleaner.api;
 
 import java.io.Serializable;
-import java.util.Collections;
-import java.util.Map;
 
 /**
  * Message that can be used to report a component usage (by the component). Consumer of this message type can
  * use the information provided to gather usage statistics/metering etc.
  */
-public class UsageMeteringMessage implements Serializable {
+public class UsageMeteringMessage implements Serializable, ComponentMessage {
+
+    private static final long serialVersionUID = 42L;
+
+    public static final char DETAILS_SEPARATOR_CHAR = ',';
+    public static final char DETAILS_QUOTE_CHAR = '"';
+
+    private static final String DETAILS_QUOTE_STR = String.valueOf(DETAILS_QUOTE_CHAR);
+    private static final String DETAILS_TWO_QUOTES_STR = DETAILS_QUOTE_STR + DETAILS_QUOTE_STR;
```

Comment:
Ok, let's do it consistently with metamodel, although it is surely a less frequent understanding of CSV.
And is there a need to encapsulate the value in double quotes, then? If the quotes in the value are escaped by backslash?